sanity_check.sh

```diff
@@ -5,21 +5,22 @@ SCRIPTS='./bessctl/conf/samples'
 OUTFILE='sanity_check.out'
 
 TESTS='exactmatch.bess
-	flowgen.bess
-	generic_encap.bess
-	hash_lb.bess
-	igate.bess
-	iplookup.bess
-	l2_forward.bess
-	multicore.bess
-	queue.bess
-	roundrobin.bess
-	s2s.bess
-	tc/complextree.bess
-	unix_port.bess
-	update.bess
-	vlantest.bess
-	wildcardmatch.bess'
+flowgen.bess
+generic_encap.bess
+hash_lb.bess
+igate.bess
+iplookup.bess
+l2_forward.bess
+multicore.bess
+queue.bess
+reattachment.bess
```

Comment:
Perhaps a bit off the topic, but we should be careful to add files here. This sanity test takes about half of the total Travis build time, being the most time consuming step (and keeps growing).

Given that what this script does is mostly identical to `wildcardmatch.bess` below, how about not adding this file, until we derive a solution for the sanity check to shorten its test time?